server.js

```diff
@@ -93,6 +105,55 @@ app.get("/accessibilite", (_, res) => res.redirect("/#/pages/accessibilité"));
 app.get("/mentions-legales", (_, res) => res.redirect("/#/pages/mentions-légales"));
 app.get("/stats", (_, res) => res.redirect("/#/stats"));
 
+// Versions
+const versionsDir = "./versions";
+let availableVersions = [];
+
+// Loading existing versions in memory
+if (fs.existsSync(versionsDir)) {
+  const dirs = fs.readdirSync(versionsDir);
+  for (const dir of dirs) {
+    const foodNoDetails = path.join(versionsDir, dir, "data/food/processes.json");
+    const textileNoDetails = path.join(versionsDir, dir, "data/textile/processes.json");
+
+    const foodDetailed = path.join(versionsDir, dir, "data/food/processes_impacts.json");
+    const textileDetailed = path.join(versionsDir, dir, "data/textile/processes_impacts.json");
+
+    const foodDetailedEnc = path.join(versionsDir, dir, "processes_impacts_food.json.enc");
+    const textileDetailedEnc = path.join(versionsDir, dir, "processes_impacts_textile.json.enc");
```

Comment:
that's a lot of repeated `path.join(versionsDir, dir`, maybe we should factor this out